src/it/java/org/jboss/tools/intellij/openshift/BaseTest.java

```diff
@@ -11,29 +11,69 @@
 package org.jboss.tools.intellij.openshift;
 
 import com.intellij.openapi.project.Project;
-import com.intellij.testFramework.fixtures.CodeInsightTestFixture;
```

Comment:
You removed all CLI test that we use to make sure new version of Odo don't break. Please restore them and use another class as parent for your new UI tests